docroot/profiles/custom/sitenow/sitenow.profile

```diff
@@ -244,6 +202,7 @@ function sitenow_preprocess_block(&$variables) {
             $publish_options = $node->get('field_publish_options')->getValue();
             if (array_search('title_hidden', array_column($publish_options, 'value')) !== FALSE) {
               // For uiowa_bootstrap/classy default.
+              // @todo Can this be removed?
               $variables["title_attributes"]['class'][] = 'element-invisible';
```

Comment:
@joewhitsitt or @bspeare Can you weigh in?

Reply:
When I removed `case 'field_block:node:page:title':`, it cause the title to show up on the homepage, so I restored it. The other part was able to be removed.